hypothesis/abstact.js

```diff
@@ -13,27 +15,30 @@ AbstactStudentT.prototype.testValue = function () {
 AbstactStudentT.prototype.pValue = function () {
   const t = this.testValue();
 
-  switch (this._options.alternative) {
+  const alternative = this._one ? this._options.alternative : -this._options.alternative;
```

Comment:
I don't understand why, but I have to do this otherwise the p-values are swapped between "greater" and "less" in two-sample cases.

Reply:
It is because what you compute is the complementary cdf, "1 - stdtr(k,t) = 0.5 * incbet( k/2, 1/2, z )". You need to compute `1 - 0.5 * cephes.incbet( k/2, 1/2, z )`.

Reply:
If I compute `1 - 0.5 * cephes.incbet( k/2, 1/2, z )`, the problem is just reversed. What I don't understand is that I need to compute the complementary cdf in one case (one-sample ttest) and not in the other case (two-sample ttest)